README.md

```diff
@@ -1,70 +1,46 @@
-# Getting Started with Create React App
+# NC NEWS APP
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is the front end I have created as part of Northcoders Coding Bootcamp.
 
-## Available Scripts
+The purpose of the project was to create an interface that allows the user to interact with the API I created earlier on the course.
 
-In the project directory, you can run:
+The app allows the user to:
 
-### `npm start`
+- view a list of all the articles stored in the API.
+- click on an article to read the full contents.
+- the user can sort the articles by author, date written,
+- votes or topic in either ascending or descending order.
+- view a page of articles on a single topic, which would also
+- allow the user to share a URL.
+- comment on an article and delete other comments they have - previously made
+- Up vote or Down vote an article (limited to one press until refreshed)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+The app was created in React and written in Vscode. The whole project was managed using Trello.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+to see the app live visit:
 
-### `npm test`
+also to see the backend I created: https://ojs-news-app1.herokuapp.com/api
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+---
 
-### `npm run build`
+## Step by step.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+to get the app running first fork and then clone the repo to your local machine.
```

Comment:
include the command here (I know most people will know how but still useful) e.g. git clone 'link'